http/cookie.ts

```diff
@@ -57,7 +57,7 @@ function toString(cookie: Cookie): string {
     out.push("HttpOnly");
   }
   if (typeof cookie.maxAge === "number" && Number.isInteger(cookie.maxAge)) {
-    assert(cookie.maxAge > 0, "Max-Age must be an integer superior to 0");
+    assert(cookie.maxAge >= 0, "Max-Age must be a non-negative integer");
```

Comment:
Minor nit on the error message:
```suggestion
    assert(cookie.maxAge >= 0, "Max-Age must be an integer superior or equal to 0");
```